src/metrics/misc.ts

```diff
@@ -194,6 +209,33 @@ export class MetadataMetric implements Metric {
       throw new Error('Invalid result');
     }
 
+    // openings
+    const opening = metadata?.find((item) => item.startsWith('[Opening "'))
+    ?.replace('[Opening "', '')
+    ?.replace('"]', '');
```

Comment:
We're stripping a lot of the metadata/repeating find in a few functions. Would it make sense to just clean it once when ingesting?

Reply:
potentially, but don't need to prioritize this rn (unless u want to). if we clean all metadata for all games that also may be inefficient (will be a lot of unused metadata). could clean the metadata for the things we specifically want (e.g. ratings, players gameType, etc) but in current form it's being done already. but since centralizing for readability is impt for u go for it